From 017e095411dc6782a5c1728c33b6957ebfd420a9 Mon Sep 17 00:00:00 2001
From: Aaron Kling <webgeek1234@gmail.com>
Date: Mon, 20 Aug 2018 22:28:06 +0200
Subject: [PATCH 7/8] Revert "releasetools: Replace key values in permission
 files during re-signing"

This reverts commit 2b673a45f94ff096aba8db7aa2cecfcf25f15b64.

This appears to no longer be used and is breaking aosp signing

Change-Id: Ide78cb4b62931ce8b52508f29cbe674c6dac5675
---
 tools/releasetools/sign_target_files_apks.py | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/tools/releasetools/sign_target_files_apks.py b/tools/releasetools/sign_target_files_apks.py
index 6b80a76ff..fa62c8f63 100755
--- a/tools/releasetools/sign_target_files_apks.py
+++ b/tools/releasetools/sign_target_files_apks.py
@@ -297,10 +297,6 @@ def ProcessTargetFiles(input_tf_zip, output_tf_zip, misc_info,
       print("Rewriting %s with new keys." % (info.filename,))
       new_data = ReplaceCerts(data)
       common.ZipWriteStr(output_tf_zip, out_info, new_data)
-    elif info.filename.startswith("SYSTEM/etc/permissions/"):
-      print("rewriting %s with new keys." % info.filename)
-      new_data = ReplaceCerts(data)
-      common.ZipWriteStr(output_tf_zip, out_info, new_data)
 
     # Ask add_img_to_target_files to rebuild the recovery patch if needed.
     elif info.filename in ("SYSTEM/recovery-from-boot.p",
-- 
2.17.1

